src/hooks/core/chat/useDMMessage.ts

```diff
@@ -0,0 +1,144 @@
+import {v4 as uuid} from 'uuid';
+import useLocalDatabaseHook from '../../../database/hooks/useLocalDatabaseHook';
+import ChannelListMemberSchema from '../../../database/schema/ChannelListMemberSchema';
+import ChannelList from '../../../database/schema/ChannelListSchema';
+import ChatSchema from '../../../database/schema/ChatSchema';
+import UserSchema from '../../../database/schema/UserSchema';
+import {initChatFromPost, initChatFromPostAnon} from '../../../service/chat';
+import {getChannelListInfo} from '../../../utils/string/StringUtils';
+import useUserAuthHook from '../auth/useUserAuthHook';
+import useChatUtilsHook from './useChatUtilsHook';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

These imports are from different directories. It's a good practice to group and separate them by directory for better readability.
```diff
- import {initChatFromPost, initChatFromPostAnon} from '../../../service/chat';
- import {getChannelListInfo} from '../../../utils/string/StringUtils';
- import useUserAuthHook from '../auth/useUserAuthHook';
- import useChatUtilsHook from './useChatUtilsHook';
+ import useUserAuthHook from '../auth/useUserAuthHook';
+ import useChatUtilsHook from './useChatUtilsHook';
+ import {initChatFromPost, initChatFromPostAnon} from '../../../service/chat';
+ import {getChannelListInfo} from '../../../utils/string/StringUtils';
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->